packages/create-cloudflare/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "create-cloudflare",
-	"version": "2.31.1",
+	"version": "2.31.2",
```

Comment:
Can you revert this change? The version number will be managed by the changeset release process.